src/Driver/Connection.php

```diff
@@ -64,7 +64,7 @@ public function lastInsertId($name = null);
      *
      * @throws Exception
      */
-    public function beginTransaction();
+    public function beginTransaction(): bool;
```

Comment:
This is a breaking change that we cannot ship in a bugfix release.

Reply:
Ok, removed